packages/api-client/src/views/Request/Request.vue

```diff
@@ -143,7 +143,12 @@ const executeRequest = async () => {
 /** Handle hotkey events from the bus */
 const handleHotKey = (event: HotKeyEvents) => {
   if (event.toggleSidebar) showSideBar.value = !showSideBar.value
-  if (event.openCommandPalette) commandPaletteBus.emit()
+
+  // We prevent default on open command so we can use it on the web
```

Comment:
was using this for testing, but seems sensible its a common practice to hijack this command for search